lib/docs/data_dictionary.md

```diff
@@ -100,3 +100,25 @@ The column name of the first column contains [embedded metadata](#embedded-metad
 | modifications | List of [modifications](#modifications) |
 | ppm | ppm tolerance |
 | rt_window | Window used for in-source decay correction (min) |
+
+### PGFinder Output
+
+| Field                             | Description                                                                                   |
+|:----------------------------------|:----------------------------------------------------------------------------------------------|
+| `Metadata`                        | Metadata about input files.                                                                   |
+| `ID`                              | Peak identifier.                                                                              |
+| `RT (min)`                        | Retention Time in minutes?                                                                    |
+| `Charge`                          | ?                                                                                             |
```

Comment:
Yes for retention time!
And the `Charge` field is the different charge states at which the mass was observed. You can use this information to work back from the monoisotopic mass to the raw mass/charge ratios that the instrument actually records